react/getting_started_with_react/react_components.md

```diff
@@ -79,26 +79,26 @@ ReactDOM.createRoot(document.getElementById('root')).render(
 )
 ~~~
 
-And just like that, you've successfully imported and used your first custom-made component, congratulations!
+Remember that `<Greeting />` should be capitalized! Try using lower case for the import, function name and component and see what happens. Otherwise, just like that, you've successfully imported and used your first custom-made component, congratulations!
```

Comment:
perhaps it should be added here then. At this point JSX is explained :)

Reply:
Ah, that's actually a good point - flows even better!